test/panel.spec.ts

```diff
@@ -1,49 +1,164 @@
+    test('Should display default image panel with base64 image', async ({
+      readProvisionedDashboard,
+      gotoDashboardPage,
+    }) => {
+      /**
+       * Go To Panels dashboard e2e.json
+       * return dashboardPage
+       */
+      const dashboard = await readProvisionedDashboard({ fileName: 'e2e.json' });
+      const dashboardPage = await gotoDashboardPage({ uid: dashboard.uid });
 
-    /**
-     * Check and compare image
-     */
-    await expect(dashboardPage.getPanelByTitle('PNG').locator.getByTestId(TEST_IDS.panel.image)).toHaveScreenshot(
-      'actual-screenshot.png'
-    );
+      /**
+       * Get panel
+       */
+      const panel = new PanelHelper(dashboardPage, 'PNG');
+
+      await panel.checkIfNoErrors();
+      await panel.checkPresence();
+
+      const imageView = panel.getImageView();
+      await imageView.checkPresence();
+      await imageView.compareScreenshot('base64-screenshot.png');
+    });
+
+    test('Should display default image panel with URL image', async ({
+      readProvisionedDashboard,
+      gotoDashboardPage,
+    }) => {
+      /**
+       * Go To Panels dashboard urls.json
+       * return dashboardPage
+       */
+      const dashboard = await readProvisionedDashboard({ fileName: 'urls.json' });
+      const dashboardPage = await gotoDashboardPage({ uid: dashboard.uid });
+
+      /**
+       * Get panel
+       */
+      const panel = new PanelHelper(dashboardPage, 'Image from URL');
+
+      await panel.checkIfNoErrors();
+      await panel.checkPresence();
+
+      const imageView = panel.getImageView();
+      await imageView.checkPresence();
+      await imageView.compareScreenshot('url-screenshot.png');
+    });
+
+    test('Should display audio panel', async ({ readProvisionedDashboard, gotoDashboardPage, page }) => {
+      /**
+       * Go To Panels dashboard postgres.json
+       * return dashboardPage
+       */
+      const dashboard = await readProvisionedDashboard({ fileName: 'postgres.json' });
+      const dashboardPage = await gotoDashboardPage({ uid: dashboard.uid });
+
+      /**
+       * Panel loading...
+       */
+      await page.waitForTimeout(2500);
+
+      /**
+       * Get panel
+       */
+      const panel = new PanelHelper(dashboardPage, 'Audio');
+
+      await panel.checkIfNoErrors();
+      await panel.checkPresence();
+
+      const audioView = panel.getAudioView();
+      await audioView.checkPresence();
+    });
+
+    test('Should display video panel with poster base64 image', async ({
+      readProvisionedDashboard,
+      gotoDashboardPage,
+      page,
+    }) => {
+      /**
+       * Go To Panels dashboard urls.json
+       * return dashboardPage
+       */
+      const dashboard = await readProvisionedDashboard({ fileName: 'urls.json' });
+      const dashboardPage = await gotoDashboardPage({ uid: dashboard.uid });
+
+      /**
+       * Panel loading...
+       */
+      await page.waitForTimeout(1500);
```

Comment:
please check if panel loaded in helper via checking if loading message disappeared

Reply:
`page.waitForTimeout` was removed